documentation/proc-pages/physics-models/fusion_reactions/plasma_reactions.md

```diff
@@ -0,0 +1,236 @@
+# Fusion Reactions
+
+## Overview
+
+The most likely fusion reaction to be utilised in a power plant is the
+deuterium-tritium reaction:
+
+$$
+\mathrm{D + T} \Longrightarrow \mathrm{^{4}He + n + 17.6 \,MeV}
```

Comment:
alter reaction to the full form of D+T -> 5He* -> 4He + n?
Probably not needed, don't think people care in fusion but know people in fission always use full form with excited states

Reply:
Dont think it is needed in this case